src/components/FiltersBar/FilterModal/FilterDrawer/FilterDrawer.js

```diff
@@ -0,0 +1,233 @@
+import React from 'react';
+import './FilterDrawer.scss';
+import {
+  Box, Button,
+} from "@material-ui/core";
+import Typography from "@material-ui/core/Typography";
+import Slider from "@material-ui/core/Slider";
+import Paper from "@material-ui/core/Paper";
+import Grid from "@material-ui/core/Grid";
+import {makeStyles, withStyles} from "@material-ui/core/styles";
+import {green} from "@material-ui/core/colors";
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import ExpansionPanel from "@material-ui/core/ExpansionPanel";
+import ExpansionPanelSummary from "@material-ui/core/ExpansionPanelSummary";
+import ExpansionPanelDetails from "@material-ui/core/ExpansionPanelDetails";
+import RadioGroup from "@material-ui/core/RadioGroup";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import Radio from "@material-ui/core/Radio";
+import Checkbox from "@material-ui/core/Checkbox";
+
+const ColorButton = withStyles(() => ({
+  root: {
+    color: green[50],
+    backgroundColor: green[500],
+    '&:hover': {
+      backgroundColor: green[600],
+    },
+  },
+}))(Button);
+
+const useStyles = makeStyles(theme => ({
+  roundedExpansionPanel: {
+    borderRadius: theme.shape.borderRadius,
+    '&::before': {
+      height: '0px'
+    }
+  },
+}));
+
+
+export default function FilterDrawer() {
+  const classes = useStyles();
+
+  const maxPrice = 10000;
+  const [price, setPrice] = React.useState([200, 3000]);
+  const [colorPanelExpanded, setColorPanelExpanded] = React.useState(false);
+  const [typePanelExpanded, setTypePanelExpanded] = React.useState(false);
+  const [orderPanelExpanded, setOrderPanelExpanded] = React.useState(false);
+  const [orderType, setOrderType] = React.useState('');
+
+  const selectTypes = [
```

Comment:
I think you should expirement with useReducer in this case, it may look cleaner instead of own state for each item in array.